content/applications/finance/expenses.rst

```diff
@@ -4,231 +4,549 @@
+  reimbursed. If the employee paid for the expense (and should be reimbursed) select
+  :guilabel:`Employee (to reimburse)`. If the company paid directly instead (e.g. if the company
+  credit card was used to pay for the expense) select :guilabel:`Company`.
+- :guilabel:`Expense Date`: Using the calendar module, enter the date the expense was incurred. Use
+  the left and right arrows to navigate to the correct month, then click on the specific day to
+  enter the selection.
+- :guilabel:`Bill Reference`: If there is any reference text that should be included for the
+  expense, enter it in this field.
+- :guilabel:`Account`: Select the expense account that this expense should be logged on from the
+  drop-down menu.
+- :guilabel:`Employee`: Using the drop-down menu, select the employee this expense is for.
+- :guilabel:`Customer to Reinvoice`: If the expense is something that should be paid for by a
+  customer, select the customer that will be invoiced for this expense from the drop-down menu. For
+  example, if an interior design customer wishes to have an on-site meeting, and agrees to pay for
+  the expenses associated with it (travel, hotel, meals, etc.), all expenses for that would indicate
+  the customer as the :guilabel:`Customer to Reinvoice`.
+- :guilabel:`Analytic Account`: Select the account the expense should be written against from the
+  drop-down menu.
+- :guilabel:`Company`: If multiple companies are set-up, select the company this expense should be
+  filed for from the drop-down menu. If there is only one company, this field will be automatically
+  populated.
+- :guilabel:`Notes...` : If any notes are needed in order to clarify the expense, enter them in the
+  notes field.
+
+ Once all the fields have been filled out, click :guilabel:`Save`.
+
+Attach a receipt
+~~~~~~~~~~~~~~~~
+
+After the expense is saved, the next step is to attach a receipt. A new :guilabel:`Attach Receipt`
+button appears after the entry is saved, beneath the former :guilabel:`Save` button (which turns
+into an :guilabel:`Edit` button).
+
+.. image:: expenses/save_receipt.png
+   :align: center
+   :alt: Attach a receipt after saving the record.
+
+Click the new :guilabel:`Attach Receipt` button, and a file explorer appears. Navigate to the
+receipt to be attached, and click :guilabel:`Open`. A new :guilabel:`Receipts` smart button appears
+at the top, and the receipt appears in the chatter. More than one receipt can be attached to an
+individual expense, if needed. The number of receipts attached to the expense will be noted on the
+smart button.
+
+.. image:: expenses/receipt_smartbutton.png
+   :align: center
+   :alt: Attach a receipt after saving the record.
+
+Automatically create new expenses from an email
+-----------------------------------------------
+
+Instead of individually creating each expense in the :guilabel:`Expenses` app, expenses can be
+automatically created by sending an email to an email alias.
```

Comment:
no menuselection needed here

```suggestion
Instead of individually creating each expense in the *Expenses* app, expenses can be automatically
created by sending an email to an email alias.
```